andela/mrm_api
# bug(prevent-non-Andela-emails): Accept only Andela emails

#### What does this PR do?
- Restricts converge users to only add Andela emails

#### Description of Task to be completed?
- Validate the email provided by the user to ensure that it has the Andela domain name

#### How should this be manually tested?
- Clone the branch
- Run make build
- Run the following mutation:
````
mutation {
  createUser(email: "newuser@gmail.com"
            name: "newuser"
            picture: "www.andela.com/user"){
    user {
      email,
      name,
      picture
    }
  }
}
````
Expected response
````
{
  "errors": [
    {
      "message": "Can't add a none Andela email",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "createUser"
      ]
    }
  ],
  "data": {
    "createUser": null
  }
}
````
#### What are the relevant pivotal tracker stories?
[#163619440](https://www.pivotaltracker.com/story/show/163619440)

#### Screenshots
![image](https://user-images.githubusercontent.com/39625835/52116710-e2f59680-2622-11e9-8788-8dd2641a8ed3.png)
